optaweb-vehicle-routing-docs/src/main/asciidoc/development-guide.adoc

```diff
@@ -132,6 +132,30 @@ https://docs.spring.io/spring-boot/docs/current/maven-plugin/reference/html/#run
 You can learn more about configuring a Spring Boot application on the
 https://docs.spring.io/spring-boot/docs/current/reference/htmlsingle/#boot-features-external-config[Spring Boot Externalized Configuration] page.
 
+The backend has a strictly enforced code style. Code formatting is done by the Eclipse code formatter, using the config files
+found in the `ide-configuration` directory. By default when you run `./mvnw install` the code will be formatted automatically.
+When submitting a pull request the CI build will fail if running the formatter results in any code changes, so it is
+recommended that you always run a full Maven build before submitting a pull request.
+
+If you want to run the formatting without doing a full build, you can run `./mvnw process-sources`.
+
+==== Eclipse Setup
+
+Open the *Preferences* window, and then navigate to _Java_ -> _Code Style_ -> _Formatter_. Click _Import_ and then
+select the `eclipse-format.xml` file in the `ide-configuration` directory.
+
+Next navigate to _Java_ -> _Code Style_ -> _Organize Imports_. Click _Import_ and select the `eclipse.importorder` file.
+
+==== IDEA Setup
+
+Open the _Preferences_ window (or _Settings_ depending on your edition) , navigate to _Plugins_ and install the [Eclipse Code Formatter Plugin](https://plugins.jetbrains.com/plugin/6546-eclipse-code-formatter) from the Marketplace.
```

Comment:
This is an Asciidoc file but you're using a Markdown-style link. Please use `URL[title]`, so something like `https://plugins...com/something[Eclipse Plugin]`.

https://asciidoctor.org/docs/asciidoc-syntax-quick-reference/#links